docs/cve-creation-guide.md

```diff
@@ -0,0 +1,189 @@
+
+### Attack vector(s)
+
+The attack vector can be difficult to determine or articulate consisely.
+For _CPANSA-Mojolicious-2018-01_ it is somewhat unclear whether the cookie for site A is leaked to Site B or simply that old cookies are sent to the web server to which you are connecting.
+
+However, it would seem that connecting to a web server may send old cookie data to that server.
+
+1. Attack vector(s): **_Connecting to a web server using a vulnerable version submits information (cookies) that should not have been submitted_**
+
+### Suggested description of the vulnerability for use in the CVE
+
+If you are using CPAN::Audit as the source information you can likely use the information listed there.
+It may also be a good idea to reference the Issue or Pull Request if either of them provide additional relevant information.
+
+
+1. Suggested description of the vulnerability for use in the CVE:
+
+   Consider that this should be useful as an email subject, and searchable.
+   The following example roughly matches the de-factor standard formatting for Perl module vulnerabilities.
+
+      **_Mojolicious before 7.66 for Perl has Cross Domain Cookie Leakage in Mojo::UserAgent::CookieJar_**
+
+   - "Mojolicious" is the name of the package and should be first in the description
+   - "before 7.66" implies that it has been fixed in later versions
+   - "for Perl" identifies this as a perl package, allowing discoverability
+   - "Cross Domain Cookie Leakage" is a short description of the vulnerability
+   - "Mojo::UserAgent::CookieJar" is the vulnerable component/function inside the package (optional)
+
+
+### Discoverer(s)/Credits
+
+On occasion, the discoverer and or credit may be disputed.
+It is likely good practice to error on the side of caution and report the facts with references.
+Sometimes a maintainer will specifically credit a reporter or discoverer but mentioning the issue as logged by ... might be best.
+In the case of _CPANSA-Mojolicious-2018-01_ the issue listed in https://github.com/mojolicious/mojo/issues/1185 was logged by @expbbc.
+
+Interestingly @expbbc did not mention security in the initial issue.
+They did however mention that it might be considered a security issue in the [Pull Request](https://github.com/mojolicious/mojo/pull/1192)
+
+1. Discoverer(s)/Credits: **_Paul Robins (@expbbc)_**
+
+### Reference(s)
+
+If you are using CPAN::Audit as the source information it provides reference URLs that can be used here.
+
+1. Reference(s):
+
+    **_https://github.com/mojolicious/mojo/pull/1192_**
+    **_https://github.com/mojolicious/mojo/issues/1185_**
+    **_https://github.com/mojolicious/mojo/commit/c16a56a9d6575ddc53d15e76d58f0ebcb0eeb149_**
+
+### Additional information
+
+You can provide additional information that you may have if it is relevant.
+
+Some possible additional information:
+1. If the vulnerability is embargoed (not publicly acknowledged or under a non disclosure of some sort) mention it here
```

Comment:
Also, maybe worth mentioning that a vuln that has been discussed in public is by default public and not embargoed.

So References should not be given for anything embargoed

Reply:
fixing